packages/testsuite/cypress/e2e/runtime/test-runtime-server-status.cy.ts

```diff
@@ -0,0 +1,167 @@
+describe("TESTS: Runtime => Server => Status", () => {
+  let managementEndpoint: string;
+  let serverHost: string;
+
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+      cy.task("execute:cli", {
+        managementApi: managementEndpoint + "/management",
+        operation: "read-attribute",
+        address: [],
+        name: "name",
+      }).then((result) => {
+        serverHost = (result as { result: string }).result;
+      });
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  beforeEach(() => {
+    cy.navigateTo(managementEndpoint, "runtime");
+    cy.get(`#standalone-host-${serverHost}`).click();
+    cy.get("#server-runtime-status").click();
+    cy.get("#hal-finder-preview").should("be.visible");
+  });
+
+  it("Read used heap", () => {
+    cy.get("#server-runtime-status-heap-used").should("be.visible");
+    cy.readAttributeAsObject(
+      managementEndpoint,
+      ["core-service", "platform-mbean", "type", "memory"],
+      "heap-memory-usage"
```

Comment:
This two parameters can be constant. They are used more then once and it will be more readable then.